example/ios/Runner/AppDelegate.m

```diff
@@ -7,7 +7,6 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application
     didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   [GeneratedPluginRegistrant registerWithRegistry:self];
-  [Radar initializeWithPublishableKey:@"prj_test_pk_0000000000000000000000000000000000000000"];
```

Comment:
For consistency, should this also be removed from the Android example code here?

https://github.com/radarlabs/flutter-radar/blob/master/example/android/app/src/main/java/io/radar/example/MainApplication.java#L12

Is the idea that `example/lib/main.dart.initRadar()` already has the call to `Radar.initialize()`?

Reply:
I checked that too, but android needs to be initialized on the java side as well. @tjulien

Reply:
Interesting.  For consistency, should we leave it in iOS as well then, and remove it from example dart lib?

Reply:
That's so weird, the actual logic are both, but in android, we have to call native initialize. Btw I updated the example @tjulien